report/renderer/category-renderer.js

```diff
@@ -144,6 +144,35 @@ export class CategoryRenderer {
     return auditEl;
   }
 
+  /**
+   * Inject the final screenshot next to the score gauce of the first category (likely Performance)
+   * @param {HTMLElement} categoriesEl
+   * @param {LH.ReportResult['audits']} audits
+   * @param {Element?} scoreScaleEl
+   */
+  injectFinalScreenshot(categoriesEl, audits, scoreScaleEl) {
+    // TODO: Use full-page-screenshot instead as that's always gathered, regardless of category filter
+    const audit = audits['final-screenshot'];
+    if (!audit || audit.scoreDisplayMode === 'error') return null;
+    const details = audit.details;
+    if (!details || details.type !== 'screenshot') return null;
+    const finalScreenshotDataUri = details.data;
+    const finalSSimg = this.dom.createElement('img', 'lh-final-ss-image');
+    finalSSimg.src = finalScreenshotDataUri;
+
+    const firstCatHeaderEl = this.dom.find('.lh-category .lh-category-header', categoriesEl);
+    const leftColEl = this.dom.createElement('div', 'lh-category-headercol');
+    const seperatorEl = this.dom.createElement('div',
+        'lh-category-headercol lh-category-headercol--seperator');
+    const rightColEl = this.dom.createElement('div', 'lh-category-headercol');
+
+    leftColEl.append(...firstCatHeaderEl.childNodes);
+    if (scoreScaleEl) leftColEl.append(scoreScaleEl);
+    rightColEl.append(finalSSimg);
+    firstCatHeaderEl.append(leftColEl, seperatorEl, rightColEl);
+    firstCatHeaderEl.classList.add('lh-category-header__finalss');
```

Comment:
at first though this was a typo: `finalss`. but I suppose it means `screenshot`? i'd suggest just spelling it all out to be clear

Reply:
I'm unresolving because of mixed signals: you've liked it, hit Resolve, but haven't changed anything :)

Reply:
no prob. i was resolving yesterday and thought i'd push then, but i'm pushing now.